### PR TITLE
fix issue #782

### PR DESCRIPTION
**Pull request by varisd**
Issue #782 showed that there were some problems with edge cases (empty sentences, etc.). This PR solves these issues
test_chrf.py is also included to automatically test desired behaviour

### CODE REVIEW

File: neuralmonkey/tests/test_chrf.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3.5
+
+
+import unittest
+
+from neuralmonkey.evaluators.chrf import ChrFEvaluator
+
+
+CORPUS_DECODED = [
```
Comment (jlibovicky) [inline neuralmonkey/tests/test_chrf.py]: Neni lepší to naimportovat z testu na BLEU?

File: neuralmonkey/evaluators/chrf.py [hunk 2]
```diff
@@ -58,44 +58,39 @@ def score_instance(self,
...
 
     def _get_ngrams(self, tokens: List[str], n: int) -> NGramDicts:
-        if len(tokens) < n:
-            self.max_ord = len(tokens)
-
         ngr_dicts = []
         for m in range(1, n + 1):
             ngr_dict = {}  # type: Dict[str, int]
-            for i in range(m, len(tokens)):
+            # if m > len(tokens), return an empty dict
```
Comment (jlibovicky) [inline neuralmonkey/evaluators/chrf.py]: Komentář vyfičet.

File: neuralmonkey/tests/test_chrf.py (hunk 1 above)
Comment (varisd) [reply neuralmonkey/tests/test_chrf.py]: Jasne, to asi jo. Nebo respektive to mit uplne bokem; porovnavani s hypo vet s refe vetama delas u vice metrik
